commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/PaperCommandRegistration.java

```diff
@@ -17,17 +26,59 @@
 public class PaperCommandRegistration<Source> extends CommandRegistrationStrategy<Source> {
 	// References to necessary methods
 	private final Supplier<CommandDispatcher<Source>> getBrigadierDispatcher;
+	private final Runnable reloadHelpTopics;
 	private final Predicate<CommandNode<Source>> isBukkitCommand;
 
 	// Store registered commands nodes for eventual reloads
 	private final RootCommandNode<Source> registeredNodes = new RootCommandNode<>();
 
-	public PaperCommandRegistration(Supplier<CommandDispatcher<Source>> getBrigadierDispatcher, Predicate<CommandNode<Source>> isBukkitCommand) {
+	private static final Object paperCommandsInstance;
+	private static final Field dispatcherField;
+
+	private static final Constructor<?> pluginCommandNodeConstructor;
+	private static final SafeVarHandle<CommandNode<?>, Object> metaField;
+
+	static {
+		Object paperCommandsInstanceObject = null;
+		Field dispatcherFieldObject = null;
+
+		try {
+			paperCommandsInstanceObject = Class.forName("io.papermc.paper.command.brigadier.PaperCommands").getField("INSTANCE").get(null);
+			dispatcherFieldObject = Class.forName("io.papermc.paper.command.brigadier.PaperCommands").getDeclaredField("dispatcher");
+		} catch (ReflectiveOperationException e) {
+			// Doesn't happen, or rather, shouldn't happen
+		}
+
+		paperCommandsInstance = paperCommandsInstanceObject;
+		dispatcherField = dispatcherFieldObject;
+		dispatcherField.setAccessible(true);
```

Comment:
Could use `CommandAPIHandler#getField` here:
```suggestion
		try {
			Class<?> paperCommands = Class.forName("io.papermc.paper.command.brigadier.PaperCommands");
			paperCommandsInstanceObject = paperCommands.getField("INSTANCE").get(null);
			dispatcherFieldObject = CommandAPIHandler.getField(paperCommands, "dispatcher");
		} catch (ReflectiveOperationException e) {
			// Doesn't happen, or rather, shouldn't happen
		}

		paperCommandsInstance = paperCommandsInstanceObject;
		dispatcherField = dispatcherFieldObject;
```